### PR TITLE
Index work - Review- DO NOT MERGE

### PR DESCRIPTION
**Pull request by Anoninnyc**
### What github.com/wevote/WebApp/issues does this fix?
#746

### Changes included this pull request?
Index.html

### CODE REVIEW

File: Gulpfile.js [hunk 1]
```diff
@@ -161,4 +161,4 @@ gulp.task("default", [
   "clean:build",
   "watch",
   "server"
-]);
+]);
```
Comment (DaleMcGrew) [inline Gulpfile.js]: Please be careful to not make extraneous changes to files that aren't part of your core check-in. Thank you!

File: src/index.html [hunk 2]
```diff
@@ -19,7 +13,9 @@
   <body>
     <div id="app" class="app-wrapper">
       <div className="loading-screen">
-        <div className="u-loading-spinner u-loading-spinner--light" style="
+        <div 
+        className="u-loading-spinner u-loading-spinner--light" 
+        style="
```
Comment (DaleMcGrew) [inline src/index.html]: This code change is also an example of an unrelated change that should be cleaned up/removed before submitting a pull request.

File: src/index.html [hunk 3]
```diff
@@ -31,11 +27,21 @@
...
     <script src="//code.jquery.com/jquery-1.12.0.min.js"></script>
+    <script>
+       $("head").append('<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css" rel="stylesheet" />');
+       $("head").append('<link href="/css/bootstrap.css" rel="stylesheet" />');
+       $("head").append('<link href="//cdn.bootcss.com/bootstrap-social/4.12.0/bootstrap-social.min.css" rel="stylesheet" />');
+       $("head").append('<link href="//cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css" rel="stylesheet"/>');
+       $("head").append('<link href="//cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css" rel="stylesheet"/>');
+       $("head").append('<link href="//cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css" rel="stylesheet" />');
+       $("head").append('<link href="/css/main.css" rel="stylesheet" />');
+      </script>
```
Comment (DaleMcGrew) [inline src/index.html]: @Anoninnyc is there a reason you use multiple '$("head").append' entries instead of a single append?
Also, you repeat the append of "slick.min.css" twice.

**Comment by Anoninnyc** [comment]
@DaleMcGrew - to be clear- the purpose of the PR was just to show how I was appending after the fact. I didn't take the time to optimize/clean it up, as I was unsure as to whether you would approve of the general changes I've made. I'll be sure to clean this up ASAP.
